app/models/manageiq/providers/cloud_manager/template.rb

```diff
@@ -3,6 +3,7 @@ class ManageIQ::Providers::CloudManager::Template < ::MiqTemplate
 
   def self.eligible_for_provisioning
     super.where(:type => %w(ManageIQ::Providers::Amazon::CloudManager::Template
+                            ManageIQ::Providers::Azure::CloudManager::Template
                             ManageIQ::Providers::Openstack::CloudManager::Template))
```

Comment:
@matthewd in a future PR, would be good to have this list be created via registration or composition, instead of hard-coding class names.